ext/oci8/tests/bug70700.phpt

```diff
@@ -0,0 +1,180 @@
+--TEST--
+Tests for LOBs with multibyte strings
+(Doc Bug #70700)
+--CREDITS--
+Chuck Burgess
+ashnazg@php.net
+--SKIPIF--
+<?php
+$target_dbs = array('oracledb' => true, 'timesten' => false);  // test runs on these DBs
+require(dirname(__FILE__).'/skipif.inc');
+?> 
+--FILE--
```

Comment:
Yes the dot should be there.  It is the separator from the language & territory component.  In this case the language & territory are empty meaning the defaults are used.  Other examples exist in PHP tests: http://lxr.php.net/xref/PHP-MASTER/ext/oci8/tests/bug47281.phpt#17